test/common/schemaform/fields/ArrayField.unit.spec.jsx

```diff
@@ -239,4 +241,181 @@ describe('Schemaform <ArrayField>', () => {
 
     expect(tree.subTree('button').props.disabled).to.be.true;
   });
+  it('should render review mode when specified and has data', () => {
```

Comment:
Does calling it `review mode` here make sense? Seems like that could be confusing. What do you think of `view mode`?

Reply:
That's fine too, review mode is used in the field itself, also, this is sort of the array equivalent of the review card field now, so that was probably why I used "review" here.